data/_redirects

```diff
@@ -1 +1,8 @@
 /blog/author  /blog
+
+/people/*  /people#:splat
+/case-studies/*  /project-stories
+/about-us  /contact
+/jobs/tester  /jobs
+/jobs/developer  /jobs
```

Comment:
I think it might be better to set these jobs ones as 302, you should be able to do that with the line like `/jobs/developer/ /jobs 302`